libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Visualizations/FunnelChart/FunnelChartScenario.tsx

```diff
@@ -0,0 +1,17 @@
+// (C) 2007-2023 GoodData Corporation
+import React from "react";
+import { FunnelChart } from "@gooddata/sdk-ui-charts";
+import {
+    Amount,
+    Department,
+} from "../../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_tiger";
+
+const style = { height: 300 };
+
+export const FunnelChartScenario: React.FC = () => {
+    return (
+        <div style={style} className="s-funnel-chart">
+            <FunnelChart measures={[Amount, Department]} />
```

Comment:
We should create a Funnel chart with 1 measure + 1 attribute, not 2 measures like above